vmaas/load.go

```diff
@@ -344,6 +345,13 @@ func loadPkgDetails(c *Cache) {
 		nevra2id[nevra] = pkgID
 
 		if det.SrcPkgID == nil {
+			// `pkgID` is the source package id
+			if _, ok := srcPkgNameID2SrcPkgIDs[det.NameID]; !ok {
+				srcPkgNameID2SrcPkgIDs[det.NameID] = []PkgID{}
+			}
+			srcPkgNameID2SrcPkgIDs[det.NameID] = append(srcPkgNameID2SrcPkgIDs[det.NameID], pkgID)
```

Comment:
yes and that is expected and part of the solution since we get only a package name from CSAF VEX file and we have no idea if that name is a name of a certain package or a name of a source package which makes all packages built from that package name affected.
sure different rhel version need to be filtered using the CPE (which we don't know here, only later during load of CSAF data), having packages for architectures there is correct as architecture is not part of CPE